webclient/components/pt-info/single/common/show-content-in-card-component.vue

```diff
@@ -28,7 +28,6 @@
             @mouseenter="
               mfHandleNewMouseEnterEventInCardHeader(singleCardHeaderAction.actionDescription)
```

Comment:
if you remove mouseOut the user can go to timeline icon and then go back to header. The timeline icon will stay active.

Reply:
Ok I am checking it and fix the issue.